package.json

```diff
@@ -78,6 +78,6 @@
         "test:vagrantProduction": "vagrant ssh -c 'cd /home/vagrant/sync/universal; DISPLAY=:0; ./scripts/vagrantCloudBasedContainers.sh'",
         "posttest": "node node_modules/nyc/bin/nyc.js report -r text-summary -r html --report-dir reports --temp-directory coverage",
         "start": "node gpii.js",
-        "postinstall": "node scripts/browserifyTestDependency.js && node scripts/convertPrefs.js testData/preferences/ build/dbData/ && node scripts/convertPrefs.js tests/data/preferences/ build/tests/dbData/"
+        "postinstall": "node scripts/browserifyTestDependency.js && node scripts/convertPrefs.js testData/preferences/ build/dbData/snapset/ snapset && node scripts/convertPrefs.js testData/preferences/ build/dbData/user/ user && node scripts/convertPrefs.js tests/data/preferences/ build/tests/dbData/ user"
```

Comment:
Can you update [this README section](https://github.com/GPII/universal/blob/master/README.md#convert-preferences-data) to add which set of data are converted to what prefs type and why? Thanks.

Reply:
Okay, but I'm not sure about everything.  Here's what I know and where I'm unclear:
1. `%universal/testData/preferences/` are converted into a set of "snapset" PrefsSafes and their associated GPII keys and placed in the `%universal/build/dbData/snapset/` folder.  These are used to update the snapset preferences in CouchDB used with a running GPII.
1. `%universal/testData/preferences/` are also converted but into a set of "user" PrefsSafes and GPII keys, and placed in the `%universal/build/dbData/user/` folder.  I can't remember what these are for.
1. `%universal/tests/data/preferences/` are converted into a set of "user" PrefsSafes and GPII keys and placed in the `%universal/build/tests/dbData`/ folder.  These are used during testing with PouchDB

The main problem is number 2, but am I right about everything else?

Reply:
Number 2 is also for running integration tests with PouchDB. Both data sets from `build/dbData/user/` and `build/tests/dbData` are loaded by [PouchTestCaseHolder|https://github.com/GPII/universal/pull/626/files#diff-9a09cbe02e1a616b46e9ff110b1b1452R48].